tests/core/test_core.py

```diff
@@ -30,8 +30,10 @@ def controller(self, mocker) -> None:
         self.theme = 'default'
```

Comment:
The area section in the commit title is unsorted.

Reply:
We're not really consistent with this, but in terms of the other commits I agree this makes it clearer.